src/locale/locales/fr/messages.po

```diff
@@ -185,7 +187,7 @@ msgstr "<0>{0}</0> {1, plural, one {abonnement} other {abonnements}}"
 
 #: src/screens/StarterPack/Wizard/index.tsx:478
 msgid "<0>{0}</0> is included in your starter pack"
-msgstr ""
+msgstr "<0>{0}</0> est inclus dans votre pack de démarrage"
```

Comment:
Again, see earlier to know why we should use `fait partie` instead.

```suggestion
msgstr "<0>{0}</0> fait partie dans votre pack de démarrage"
```

Reply:
Here, should it be `fait partie dans` or `fait partie de`?